src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/ConstructedEETypeNode.cs

```diff
@@ -121,8 +121,8 @@ public static bool CreationAllowed(TypeDesc type)
 
                 default:
                     // Generic definition EETypes can't be allocated
```

Comment:
Delete this commented-out code?

Reply:
I'm not particularly fond of having to do this, hence the Draft mode and commented out code. This is the reason why it's Draft.